tests/robot/OBSERVATION/01_create.robot

```diff
@@ -820,11 +820,11 @@ Force Tags              observation_create    create
 	...                 3. *UPDATE* ``Subject - Identifier - value`` with the _UUID:_ ${subject_id} which was created in EHR record\n\n
     ...                 4. *POST* example JSON to observation endpoint\n\n
 	...                 5. *VALIDATE* the response status
-	[Tags]             	blood-pressure    valid    alternative   loinc    instant    not-ready_bug    not-ready    194
+	[Tags]             	blood-pressure    invalid    instant not-ready_bug
```

Comment:
@subigre not-ready_bug is a sub-tag of not-ready, so can be removed as well when the test  pass.
More over it'd be good to let fixed tests succeed at least once on the CI before removing the issue ID (i.e. 194). This way it's easy to spot whether fixed test work again. The issue IDs don't hurt anybody and can be removed later at any time.
![image](https://user-images.githubusercontent.com/64201310/125806994-892c7c43-fcc6-4e2a-9148-311db11af385.png)